.github/workflows/push_docker_image.yml

```
@@ -54,13 +54,15 @@ jobs:
             -f ${{ matrix.docker.dockerfile }}.amd64 \
             -t ${{ matrix.docker.repo }}:git-${{ github.sha }}-amd64 \
             --build-arg COMMIT=git-${{ github.sha }}
+            --build-arg TARGETPLATFORM=linux/amd64
```

Oops could you add backslash to treat a new line as an option of the `docker build` command?
```suggestion
            --build-arg COMMIT=git-${{ github.sha }} \
            --build-arg TARGETPLATFORM=linux/amd64
```